src/marketing/price-rules-cart-coupon.md

```diff
@@ -4,61 +4,61 @@ title: Coupon Codes
 
 Coupons codes are used with [cart price rules]({% link marketing/price-rules-cart.md %}) to apply a discount when a set of conditions is met. For example, a coupon code can be created for a specific customer group, or for anyone who makes a purchase over a certain amount. To apply the coupon to a purchase, the customer can enter the coupon code in the cart, or possibly at the cash register of your _brick and mortar_ store. Here are a few ways that you can use coupons in your store:
 
-- Email coupons to customers
-- Produce printed coupons
-- Create in-store coupons for mobile users
+-  Email coupons to customers
+-  Produce printed coupons
+-  Create in-store coupons for mobile users
 
 Coupon codes can be sent by email, or included in newsletters, catalogs, and advertisements. The list of coupon codes can be exported and sent to a commercial printer. You can also create in-store coupons with a quick response code that shoppers can scan with their smart phones. The QR code can link to a page on your site with more information about the promotion.
 
+If [dotdigital]({% link marketing/engagement-cloud.md %}) is enabled, you can use the _dotdigital Coupon URL Builder_ to dynamically generate personalized URLs with coupon codes that can be used in your dotdigital email campaigns. You can also resend a previously generated coupon code to the same contact. To learn more, see [Dynamic content: Coupon codes - Magento 2][1] in the dotdigital Help Centre.
+
 ## Method 1: Create a Specific Coupon
 
 1. Follow the instructions to create a [cart price rule]({% link marketing/price-rules-cart.md %}).
 
 1. On the **Rule Information** section, set **Coupon** to `Specific Coupon`.
 
-   - Enter a **Coupon Code** to be used with the promotion.
+   -  Enter a **Coupon Code** to be used with the promotion.
```

Comment:
is this the next step?  should it be numbered?

Reply:
Fixed. Thanks!